src/main/java/cinema/Main.java

```diff
@@ -22,6 +23,10 @@ public static void main(String[] args) {
         movie.setTitle("Fast and Furious");
         MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         movieService.add(movie);
+        Movie movie1 = new Movie();
+        movie1.setTitle("Lord of the rings");
+        movie1.setDescription("hobbits and elfs");
```

Comment:
Orks and Humans: Are we joke for you?

Reply:
Sauron lost. They are not honoured to be mentioned :)